lua/harpoon/init.lua

```diff
@@ -8,22 +8,32 @@ local cache_config = string.format("%s/harpoon.json", data_path)
 local M = {}
 
 --[[
+-- type Layout = "float" | "split" | "vsplit" | "hsplit" | "replace" (default) | "tab"
```

Comment:
@polarmutex I am thinking that this would allow for every option.

Reply:
do we need spilt with hsplit and vsplit? or what is the difference between them?